modules/mongodb/mongodb.go

```diff
@@ -83,5 +112,14 @@ func (m *Mongo) Cleanup() {
 }
 
 func (m *Mongo) initCharts() (*module.Charts, error) {
```

Comment:
```suggestion
func (m Mongo) initCharts() (*module.Charts, error) {
```

I see we don't manipulate the receiver value.

Reply:
done